src/__test__/components/data-exploration/CellInfo.test.jsx

```diff
@@ -30,24 +23,13 @@ describe('CellInfo', () => {
     };
 
     const component = mount(
-      <Provider store={store}>
-        <CellInfo componentType='heatmap' coordinates={coordinates} />
-      </Provider>,
+      <CellInfo componentType='heatmap' coordinates={coordinates} cellInfo={cellInfo} />,
     );
 
     expect(component.find(Card).length).toEqual(1);
   });
 
   test('does not show when hovering over different component', () => {
```

Comment:
If we don't need the `componentType` param and component check, this test is not needed